### ISSUE TITLE
Clarification on CustomMetrics needed

### ISSUE THREAD

**Issue by Leonardo-Ferreira** [microsoft/ApplicationInsights-dotnet#2007]
* List of [NuGet packages](https://github.com/microsoft/ApplicationInsights-dotnet#nuget-packages) and version that you are using:
```
    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.14.0" />
    <PackageReference Include="Microsoft.ApplicationInsights.DependencyCollector" Version="2.14.0" />
    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.14.0" />
```
* Runtime version (e.g. net461, net48, netcoreapp2.1, netcoreapp3.1, etc. You can find this information from the `*.csproj` file):
`<TargetFramework>netcoreapp3.1</TargetFramework>`
* Hosting environment (e.g. Azure Web App, App Service on Linux, Windows, Ubuntu, etc.):
`FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-bionic`

**Question:**

On  the`customMetrics` values (captured automatically by the sdk) there are `valueCount`, `valueSum`, `valueMin`, `valueMax` and `valuesStdDev` columns... but in my cases `valueCount` is always `1` and value=valueSum=valueMin=valueMax...

so, what is the propose of such columns? am I doing something wrong? (like, reading the info wrongly)

Also, for some metrics like `System.Runtime|Exception Count` "inconsistent" values (such as 4.45) are being reported. I mean, you either have a exception or you don't...

**Comment by cijothomas** [comment]
Can you give a specific example of customMetric values captured automatically by the SDK -?
Also when you say "my case" - which API did you use to send metric? TrackValue() or Getmetric.Track() ?

**Comment by Leonardo-Ferreira** [comment]
The metrics I'm talking about are the one that starts with "System.Runtime|", such as:

System.Runtime\|CPU Usage
System.Runtime\|Working Set
System.Runtime\|GC Heap Size
System.Runtime\|Gen 0 GC Count
System.Runtime\|Gen 1 GC Count
System.Runtime\|Gen 2 GC Count
System.Runtime\|Exception Count
System.Runtime\|ThreadPool Thread Count
System.Runtime\|Monitor Lock Contention Count
System.Runtime\|ThreadPool Queue Length
System.Runtime\|ThreadPool Completed Work Item Count
System.Runtime\|% Time in GC since last GC
System.Runtime\|Gen 0 Size
System.Runtime\|Gen 1 Size
System.Runtime\|Gen 2 Size
System.Runtime\|LOH Size
System.Runtime\|Allocation Rate
System.Runtime\|Number of Assemblies Loaded
System.Runtime\|Number of Active Timers
HeartbeatState

**Comment by pinkfloydx33** [comment]
I just stumbled here because I noticed my AI is flooded with tens of thousands of these `System.Runtime|` custom metrics and this happened to be the top search result. Does anyone know what's logging these and how they can be disabled? After doing a quick debug, it *appears* like it might be the `EventCounterCollectionModule`, can someone verify that? I've already removed the `PerformanceCollectorModule` so I'm sure it's not that.
